R/zzz.R

```diff
@@ -0,0 +1,33 @@
+#' Interact with the Python Main Module
+#' 
+#' The `py` object provides a means for interacting
+#' with the Python main session directly from \R.
+#'
+#' @format An \R object acting as an interface to the
+#'   Python main module.
+#' 
+#' @export
+"py"
```

Comment:
I think it would be preferable if we made this available only during execution of R chunks in knitr. I have often used `py <- import_builtins()` in example code so exporting `py` might conflict with this.  I also don't think that `library(reticulate)` should be required in order to access Python chunks.  Do you think this is reasonable or do you feel strongly about making it globally available?